### PR TITLE
added missing 'import logging'

### PR DESCRIPTION
**Pull request by DaveBerkeley**
Fix for logger not imported in yosys.py issue #189

### CODE REVIEW

File: edalize/yosys.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
+
```
Comment (imphil) [inline edalize/yosys.py:1]: Can you remove this empty line?

**Review by imphil — COMMENTED** [review]
Thanks for this patch! One small change and we're ready to get this merged. Please rebase the existing commit and force-push, if possible.

File: edalize/yosys.py (hunk 1 above)
Comment (DaveBerkeley) [reply edalize/yosys.py:1]: why is this necessary?

Comment (imphil) [reply edalize/yosys.py:1]: Thank you for your update to this pull request. You removed an empty line further down in the file, which is fine. However, the first line in this file is still an empty line, please remove it as well (it's unnecessary).

Comment (DaveBerkeley) [reply edalize/yosys.py:1]: what is the problem with it?

**Comment by olofk** [comment]
Thanks for the patch. Squashed and merged!

**Pull request closed without merging**